os_migrate/playbooks/delete_conversion_hosts.yml

```diff
@@ -20,6 +20,67 @@
 
 - hosts: migrator
   tasks:
+
+    - name: delete the src default conversion host
+      ansible.builtin.include_role:
+        name: os_migrate.os_migrate.conversion_host
+        tasks_from: delete.yml
+      vars:
+        os_migrate_conversion_host_name: "os_migrate_conv"
```

Comment:
We need to make sure os_migrate_conv are removed until https://github.com/os-migrate/os-migrate/pull/464 is rebased, otherwise each time the PR fails we will have resources stuck in the CI.